### PR TITLE
Add info regarding the default behaviour of the constructors with fewer args

### PR DESCRIPTION
**Pull request by IanKemp**
## Summary

As per title.

### CODE REVIEW

File: xml/System.Net.Http/HttpClient.xml [hunk 1]
```diff
@@ -194,8 +194,13 @@ public class GoodController : ApiController
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>Initializes a new instance of the <see cref="T:System.Net.Http.HttpClient" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Http.HttpClient" /> class using a <see cref="T:System.Net.Http.HttpClientHandler" /> that is disposed when this instance is disposed.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[    
+## Remarks  
+Using this constructor is equivalent to calling [HttpClient(new HttpClientHandler(), true)](xref:System.Net.Http.HttpClient(HttpHandler,boolean)).
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
Using this constructor is equivalent to calling the [`HttpClient(new HttpClientHandler(), true)`](xref:System.Net.Http.HttpClient.%23ctor(System.Net.Http.HttpMessageHandler,System.Boolean)) constructor.
```

File: xml/System.Net.Http/HttpClient.xml [hunk 2]
```diff
@@ -224,14 +229,14 @@ public class GoodController : ApiController
       </Parameters>
       <Docs>
         <param name="handler">The HTTP handler stack to use for sending requests.</param>
-        <summary>Initializes a new instance of the <see cref="T:System.Net.Http.HttpClient" /> class with a specific handler.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Http.HttpClient" /> class with the provided handler. The handler is disposed when this instance is disposed.</summary>
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Net.Http.HttpClient" /> class with the specified handler. The handler is disposed when this instance is disposed.</summary>
```

File: xml/System.Net.Http/HttpClient.xml [hunk 3]
```diff
@@ -224,14 +229,14 @@ public class GoodController : ApiController
...
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
+          <format type="text/markdown"><![CDATA[    
 ## Remarks  
- `handler` will be disposed of by calling [HttpClient.Dispose](xref:System.Net.Http.HttpMessageInvoker.Dispose).  
-  
- ]]></format>
+Using this constructor is equivalent to calling [HttpClient(handler, true)](xref:System.Net.Http.HttpClient(HttpHandler,boolean)).
```
Comment (rpetrusha) [inline xml/System.Net.Http/HttpClient.xml]:
```suggestion
Using this constructor is equivalent to calling the [`HttpClient(handler, true)`](xref:System.Net.Http.HttpClient.%23ctor(System.Net.Http.HttpMessageHandler,System.Boolean)) constructor.
```

**Comment by mairaw** [comment]
@IanKemp can you apply the suggestions made by @rpetrusha?

**Pull request merged**